.github/workflows/linters.yml

```diff
@@ -0,0 +1,19 @@
+name: Linters
```

Comment:
Instead of setting up a specific action, we can just instead enable pre-commit.ci in this repository... it not only checks that pre-commit runs, but also updates the PR with fixes when possible.

We use all over pytest-dev and in several other projects, cannot recommend enough.

Reply:
sure thing, let me add it

Reply:
@gabrielcnr will need to enable [pre-commit.ci](https://pre-commit.ci/) as he is the admin.

Reply:
It seems just @gabrielcnr can add it